src/gtk/toga_gtk/widgets/progressbar.py

```diff
@@ -7,19 +7,29 @@ class ProgressBar(Widget):
     def create(self):
         self.native = Gtk.ProgressBar()
         self.native.interface = self.interface
+        self.timeout_id = GObject.timeout_add(50, self._animate, None)
```

Comment:
Is the 50ms timeout a recommended value? It seems like a very active callback. Also - should this callback be disabled if the progress bar isn't running? No point polling if there's no update to perform.

Reply:
I got the 50ms from [this GTK3 ReadTheDocs example](http://python-gtk-3-tutorial.readthedocs.io/en/latest/progressbar.html). I suppose it could be slowed, maybe to 100-200ms or something, but this is actually animating the movement of the progress bar from one side to the other, so too slow of a timeout would appear choppy. I agree 100% that is can be disabled if unused. This change would require `start()` and `stop()` methods, which it sounds like I may be needing anyway.